README.md

```diff
@@ -130,8 +130,6 @@ Clone this repository to your desired folder:
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove the **Live Demo** section since you don't have any working live link to make your readme descriptive and professional 👍

Reply:
Thank will remove it for now 👍